src/GNDStk/child/test/child.test.cpp

```diff
@@ -0,0 +1,9 @@
+
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+#include "GNDStk.hpp"
+
+SCENARIO("Testing GNDStk child type") {
+
```

Comment:
Yes, this test is just a stub, and is there at the moment because the stipulated layout for NJOY projects wants something there.

I suppose that an issue with just about any test system is: where exactly do tests go? It's easy to say that a test of class `child_t` would go, well, right here. But, of course, code constructs rarely, if ever, exist in a vacuum. Object of type `child_t` are made throughout the code base as well as in several other tests - because those `child_t` objects are needed for the purpose of testing other constructs. In effect, `child_t` itself is also tested in those other places. I suppose I could put some simple `child_t` objects here, just testing the constructors and such, and a small amount of relatively new functionality that I have for the type, and know I'm not testing anywhere else.